posts/2021-09-21-building-on-atomic-reference.adoc

```diff
@@ -0,0 +1,293 @@
+---
+layout: post
+title: "AcidReference: building on AtomicReference"
```

Comment:
something is causing the metadata block to not render properly in github (click "View file" to see what I mean) and I think it will also cause the doc build to fail. I think it might be the colon in the title. Recommend replacing it in the `title` and `seo-title` fields with a dash.

Reply:
I have updated -- and improved -- the title.
It has no colons now.